nym-vpn-api/vpn-api-lib-wasm/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "nym-vpn-api-lib-wasm"
+version = "0.1.0"
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+bs58 = { workspace = true }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
+time = { workspace = true, features = ["wasm-bindgen"] }
+thiserror.workspace = true
+zeroize = { workspace = true }
+
+# wasm-specific deps
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
since you're using the monorepo now, you might want to use workspace deps instead

Reply:
Well these crates are not part of the workspace yet, but will be soon, hopefully after the sqlx upgrade is merged